gusto/transport_equation.py

```diff
@@ -220,11 +220,41 @@ class EmbeddedDGAdvection(AdvectionEquation):
               constructed for you.
     :arg solver_params: (optional) dictionary of solver parameters to pass to the
                         linear solver.
+    :arg recovered_spaces: A list or tuple of function spaces to be used for the
+                           recovered space advection method. The must be three
+                           spaces, indexed in the following order:
+                           [0]: the embedded space in which the advection takes place.
+                           [1]: the continuous recovered space.
+                           [2]: a broken or discontinuous version of the original space.
+                           The default for this option is None, in which case the method
+                           will not be used.
     """
 
+<<<<<<< HEAD
+    def __init__(self, state, V, ibp="once", equation_form="advective", vector_manifold=False, Vdg=None, solver_params=None, recovered_spaces=None):
+=======
     def __init__(self, state, V, ibp="once", equation_form="advective", vector_manifold=False, Vdg=None, solver_params=None, outflow=False):
-
-        if Vdg is None:
+>>>>>>> master
```

Comment:
You need to resolve the conflict in the argument list to `__init__`.  Presumably you need to union of the two argument lists now.

Reply:
Oops, sorry. Hopefully lesson learned!